test-libz-rs-sys/src/gz.rs

```diff
@@ -1022,6 +1022,154 @@ fn gzputs_error() {
     assert_eq!(unsafe { gzclose(file) }, Z_ERRNO);
 }
 
+#[test]
+fn gzgetc_basic() {
+    // Read data from a gzip file one byte at a time using gzgetc, and verify that
+    // the expected content is returned.
+    for gzgetc_fn in [|x| unsafe { gzgetc(x) }, |x| unsafe { gzgetc_(x) }] {
```

Comment:
@brian-pane can you add a fixme comment here (and below where we do something similar) linking to https://github.com/rust-lang/rust/issues/140293?